android/src/main/java/com/bitmovin/player/reactnative/RNPlayerView.kt

```diff
@@ -102,8 +103,26 @@ class RNPlayerView(
     private val activityLifecycle = (context.currentActivity as? ReactActivity)?.lifecycle
         ?: error("Trying to create an instance of ${this::class.simpleName} while not attached to a ReactActivity")
 
+    /**
+     * Relays the provided set of events, emitted by the player, together with the associated name
+     * to the `eventOutput` callback.
+     */
+    private var playerEventRelay: EventRelay<Player, Event>
+
+    private var playerInMediaSessionService: Player?
+        get() = context.playerModule?.mediaSessionPlaybackManager?.serviceBinder?.player
+        set(value) {
+            value?.let {
+                context.playerModule?.mediaSessionPlaybackManager?.serviceBinder?.player = it
+            }
+        }
+
     private val activityLifecycleObserver = object : DefaultLifecycleObserver {
+        // Don't stop the player when going to background
```

Comment:
This comment looks out of place for the onStart lifecycle.

Reply:
Yeah I agree, it is confusing.
Address in 5a4ff3c379dcaef1dab6fbaa5d6ed9349b976770